pkgs/tools/misc/btrfs-assistant/default.nix

```diff
@@ -0,0 +1,89 @@
+{ lib
```

Comment:
Can you please sort the lambda attributes ?

Reply:
I sorted it this way: https://github.com/NixOS/nixpkgs/pull/241997#discussion_r1282525140